sympy/concrete/summations.py

```diff
@@ -574,13 +576,16 @@ def eval_sum(f, limits):
 
 
 def eval_sum_direct(expr, limits):
+    from sympy.core import Add
```

Comment:
@smichr I don't know what is the motivation for these changes. I saw a 2 other PR's ( #8979, #8982 ) doing the same and I thought probably this was discussed so I sent this PR. 

Even I don't know which path is better to use. Some places people have used full paths and some other places they just used partial paths.  I tried sticking to the partial path throughout my PR (`from  sympy.foo import bar`). Probably @toolforger, @asmeurer can shed some light on the situation?